addons/stock/models/stock_picking.py

```diff
@@ -564,7 +568,7 @@ def _default_picking_type_id(self):
 
     move_type = fields.Selection([
         ('direct', 'As soon as possible'), ('one', 'When all products are ready')], 'Shipping Policy',
-        default='direct', required=True,
+        required=True,
         help="It specifies goods to be deliver partially or all at once")
```

Comment:
Maybe we could turn this into a computed field, with a compute like `record.move_type = record.group_id.move_type or record.picking_type_id.shipping_policy`? Instead of setting it manually in the onchange/create functions or in the _get_new_picking_values

Reply:
I'm afraid it's a bit more complicated: transfers can be created from other models, like SOs, and in that case they should use whatever shipping policy that object has. Only if no value is provided (e.g. when we create a transfer from scratch) then we should fallback to the picking type's shipping policy.

Reply:
When it is created from a sale order, it will be set through the procurement group, so it should still work as long as we use the group_id.move_type as default and fall back to the picking_type.move_type, no? Something like:
https://github.com/odoo-dev/odoo/commit/d4635786370c8495d8bea282291f9e845f146869